### DISCUSSION TITLE
Any Plans for implementing nodejs steams ?

### DISCUSSION THREAD

**Discussion by ShivamJoker** [prisma/prisma#8433] [Q&A] [answered]
It would be really great to have the streams support so that large sets of data can be easily manipulated, rather than loading all of them in the memory at once.

I tried looking into the docs but didn't get anything

**Comment by ryands17** [comment, marked as answer]
@ShivamJoker 👋 

We don't have it yet, but it would be great if you could add a 👍 to this request so that we can look at the priority.
https://github.com/prisma/prisma/issues/5055

A workaround as of now would be to use a combination of generators and pagination to get the desired result

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
